### ISSUE TITLE
[Integration] Add Truffle Drizzle to the BuidlBox Experience

### ISSUE THREAD

**Issue by KamesCG** [uport-project/buidlbox#30]
### Background
The Drizzle library looks like a great integration candidate for the BuidlBox.

http://truffleframework.com/docs/drizzle/getting-started

### User Story
As a Frontend React Developer I want to more easily develop smart contracts features with my distributed application frontend.

### Technical Issues
I haven't committed to integrating Drizzle, because from what I understand a separate Redux store will have to be managed inside the application. However, I could be mistaken...

```
// Import contracts
import SimpleStorage from './../build/contracts/SimpleStorage.json'
import TutorialToken from './../build/contracts/TutorialToken.json'

const options = {
  contracts: [
    SimpleStorage
  ]
}

const drizzleStore = generateStore(this.props.options)
const drizzle = new Drizzle(this.props.options, drizzleStore)
```

The BuidlBox stores blockchain integrations in the `assimilation` folder, which currently includes Redux Saga templates for `Ethers.js` and `Web3.js`

Simply put, have to think through the integration a little more, in terms of how this will be utilized, whether that's more in a `development` or `production` environment.

**Comment by mtsalenc** [comment]
This will probably require using uPort's web3 with drizzle. Truffle's [#54](https://github.com/trufflesuite/drizzle/issues/54) is related to this.
